### ISSUE TITLE
File details pane

### ISSUE THREAD

**Issue by tslaton** [jupyterlab/jupyterlab#6538]
It'd be nice if one could see statistics about files in the filebrowser, similar to the output of "Get Info" on macOS. I'm thinking the UI could look something like the *Details/Activity* pane from Google Drive pictured below:

<img width="334" alt="Screen Shot 2019-06-11 at 4 10 47 PM" src="https://user-images.githubusercontent.com/9665562/59312954-cbc75500-8c63-11e9-986d-7ea54a3db523.png">

This could be written as an extension, but probably deserves to make it into the core package if implemented well.

**Comment by tslaton** [comment]
@meeseeksdev tag "pkg:filebrowser" "type:Enhancement" "tag:Extension Idea"

**Comment by jasongrout** [comment]
And perhaps a smaller change for now is to display some information (like file size) in a tooltip.

**Comment by Tanmay-Jaiswal** [comment]
Maybe display the number of files in the folder as well. That could be helpful while monitoring datasets

**Comment by tgeorgeux** [comment]
Here's a design for a context menu option and a modal window.

Here's a design for this, let me know if there are questions or input.

![Improved](https://user-images.githubusercontent.com/22109376/71027546-5039db00-20c0-11ea-8eeb-ce8a62182845.png)

![Modal](https://user-images.githubusercontent.com/22109376/71027569-59c34300-20c0-11ea-8e4b-f39de00b86aa.png)

**Comment by ggbhat** [comment]
@tgeorgeux  I would like to work. Can I start.

**Comment by tgeorgeux** [comment]
@ggbhat Absolutely! If you need help please feel free to ring in here, or open a pull request to get eyes on your code progress.

**Comment by krassowski** [comment]
I wonder if this should be combined with RTC features about files.

@JasonWeill I see that you self-assigned here, just checking if you still plan to work on this in case if someone else would want to tackle this.

**Comment by JasonWeill** [comment]
@krassowski I'm not actively working on this issue; I can untag myself.

**Comment by krassowski** [comment]
Some implementations for VSCode in extensions (nothing in core):
- https://marketplace.visualstudio.com/items?itemName=Thinker.properties
- https://marketplace.visualstudio.com/items?itemName=brunnerh.file-properties-viewer
